chatterbot/logic/specific_response.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, chatbot, **kwargs):
         self.response_statement = Statement(text=output_text)
 
     def can_process(self, statement):
-        if statement == self.input_text:
+        if str(statement) == self.input_text:
```

Comment:
Since this will always be a statement object, maybe this should be `statement.text` instead of casting to a string?

Reply:
you are right. Since the str function returns statement.text, there is no problem if statement.text is used. I will fix it